extensions/git/src/git.ts

```diff
@@ -581,6 +585,14 @@ export class Git {
 			}
 		}
 
+		/* __GDPR__
+			"git.execDuration" : {
+				"owner": "lszomoru",
+				"duration": { "classification": "SystemMetaData", "purpose": "PerformanceAndHealth","isMeasurement": true, "comment": "Time it took to run git" },
+			}
+		*/
+		this.telemetryReporter.sendTelemetryEvent('git.execDuration', undefined, { duration: Date.now() - startTime });
```

Comment:
Should we use the performance markers so that we can track both the time it took to spawn the process and to execute the git command.

Reply:
We are currently using `startTime` to compute the duration, however `startTime` is set on line 566 which is after `this.spawn()` already returned. Doesn't this mean that the duration that we publish is missing the time that it took to spawn the child process that will execute the git command?

Reply:
oh yeah, I completely missed that and just went for the existing `startTime` - assuming it does what I want. I will push an update to capture and log both values

Reply:
pushed